flashcard.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/helper.rb'
+
+class FlashcardRunner
+  def initialize(round, deck, cards)
+    @round = round
+    @deck = deck
+    @cards = cards
+  end
+
+  def start
+    puts "Welcome! You're playing with 4 cards.
+         -------------------------------------------------
+         This is card number #{@round.turn_count + 1} out of 4."
+    ask_question
+  end
+
+  def ask_question
+    puts "Question: #{@round.current_card.question}"
+    check_input
+  end
+
+  def check_input
+    guess = gets.chomp.downcase
```

Comment:
Because `answer` on `Card` is not always downcased, this can created some incorrect results. For example, in the `cards.txt` the answer to the cardboard cutout question is "Justin bieber". If the user's answer is downcased, then it will always be stored as "justin bieber" which will always return as incorrect because `guess == answer` will never return as true if the answer isn't also stored as downcase upon creation.